packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -14,15 +14,18 @@ const SearchSuggestions = ({
   const getLinkToCourse = (course) => `/${enterpriseSlug}/course/${course.key}`;
   const getLinkToProgram = (program) => `/${enterpriseSlug}/program/${program.aggregation_key.split(':').pop()}`;
 
+  const courses = autoCompleteHits.filter(hit => hit.learning_type === 'course');
+  const programs = autoCompleteHits.filter(hit => hit.learning_type === 'program');
+  const execEdCourses = autoCompleteHits.filter(hit => hit.learning_type === 'Executive Education');
```

Comment:
[clarification] Are we intentionally not handling `pathways` as a `learning_type` here?

<img width="322" alt="image" src="https://github.com/openedx/frontend-enterprise/assets/2828721/d1ce0d12-a52a-411a-ac47-f28b22fdf783">

Reply:
Only because it doesn't really apply to my use case

Reply:
talked offline, pathways futures are tbd and don't really apply to my use case so we're holding off